beacon_node/src/cli.rs

```diff
@@ -194,6 +194,12 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .help("Lighthouse by default does not discover private IP addresses. Set this flag to enable connection attempts to local addresses.")
                 .takes_value(false),
         )
+        .arg(
+            Arg::with_name("disable-backfill-rate-limiting")
```

Comment:
Same as here, I suggest making this a flag stated in positive

Reply:
happy to keep this one as is, if we use in code a more easy to follow one

Reply:
I've renamed the config name in code